Source/WebCore/style/StyleFontSizeFunctions.cpp

```diff
@@ -183,5 +184,19 @@ int legacyFontSizeForPixelSize(int pixelFontSize, bool shouldUseFixedDefaultSize
     return findNearestLegacyFontSize<float>(pixelFontSize, fontSizeFactors, mediumSize);
 }
 
+float adjustedFontSize(float size, float sizeAdjust, float xHeight)
+{
+    float aspectValue = xHeight / size;
```

Comment:
What guarantees `size` is not zero?

What guarantees `xHeight` is not zero?

Reply:
`size` can definitely be 0, this is common in practice